dub.selections.json

```diff
@@ -0,0 +1,7 @@
+{
+	"fileVersion": 1,
+	"versions": {
```

Comment:
this file should be removed

Reply:
also, needs meson.build, travis and circle ci, but they can be added in a separate mr

Reply:
I copied over the gitignore from mir-algorithm as well.